src/components/TokenSelect/NoTokensAvailableOnNetwork.tsx

```diff
@@ -0,0 +1,31 @@
+import { Trans } from '@lingui/macro'
+import { HelpCircle } from 'icons'
+import styled, { css } from 'styled-components/macro'
+import { ThemedText } from 'theme'
+
+import Column from '../Column'
+
+const HelpCircleIcon = styled(HelpCircle)`
+  height: 64px;
+  margin-bottom: 12px;
+  stroke: ${({ theme }) => theme.secondary};
+  width: 64px;
+`
+
+const wrapperCss = css`
+  display: flex;
+  height: 80%;
+  text-align: center;
+  width: 100%;
+`
+
+export default function NoTokensAvailableOnNetwork() {
+  return (
+    <Column align="center" justify="center" css={wrapperCss}>
```

Comment:
Instead of `css`, wrap the `Column` into a new StyledComponent and use that directly.